### PR TITLE
Include drupal cache fetch (review instances)

### PR DESCRIPTION
**Pull request by jbritt1**
## Description
Adds the fetch of drupal content to review instance build process. 

## Testing done
Added the line in this PR and ran review instance build(s) locally from my laptop.

## Screenshots
image

## Acceptance criteria
- [x] Review instance building with latest drupal content. 

## Relates
- https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/6084

## Depends on
- https://github.com/department-of-veterans-affairs/devops/pull/6533 (needs to be merged before this)

## Definition of done
- [x] Events are logged appropriately - n/a
- [x] Documentation has been updated, if applicable - n/a
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by rianfowler** [comment]
wait for approval by @cvalarida

**Review by cvalarida — APPROVED** [review]
:shipit:

**Pull request merged**
